frontend/lib/more/more_page.dart

```diff
@@ -17,7 +21,8 @@ class MorePage extends StatelessWidget {
             title: 'Über diese App',
             icon: Icons.info_outline,
             callback: () => _showAboutDialog(context),
-          )
+          ),
+          SHOW_TEST_DATA_OPTIONS ? TestingDataItem() : SizedBox.shrink(),
```

Comment:
```suggestion
          if (SHOW_TEST_DATA_OPTIONS) TestingDataItem(),
```
Seems like Darts collection if could be useful here

Reply:
Nice! That was what I was looking for :laughing: